alternator/server.cc

```diff
@@ -20,7 +20,7 @@
 #include "service/client_state.hh"
 #include "service/qos/service_level_controller.hh"
 #include "utils/assert.hh"
-#include "timeout_config.hh"
+#include "../timeout_config.hh"
```

Comment:
This change isn't necessary or relevant to this patch.
We also include other stuff in the root directory, like "client_data.hh" like that, and not as "../client_data.hh".
We can old a discussion on whether we need to change this, but it should be part of this patch.

Reply:
ouch... refactoring thingy, will revert